lib/index.js

```diff
@@ -98,6 +99,11 @@ module.exports = function (gulp, options) {
 
   styleguide(gulp, lintOptions);
 
+  gulp.task('shrinkwrap', 'Shrinkwrap the existing node modules.' +
```

Comment:
Can we document this in `README.md`? Also, although it's been our practice in some of our projects, I really think `--existing` should be the default behavior; there's no reason why you should have to reinstall all modules. Plus it makes it hard to avoid unwanted upgrades when you're just trying to make one change, for example.